packages/api/resolvers/queries/users.js

```diff
@@ -1,19 +1,38 @@
 import { log } from 'meteor/unchained:core-logger';
 import { Users } from 'meteor/unchained:core-users';
 
+Users.rawCollection().createIndex({
```

Comment:
Please move this to the initializing function which also starts the migrations (see upcoming changes in https://github.com/unchainedshop/unchained/pull/229 for reference)